daisy_workflows/build-publish/README.md

```diff
@@ -11,18 +11,20 @@ These are [Daisy] and [`gce_image_publish`] workflows used by Google to build
 
 The following directories exist
 
-* bare\_metal - wrapper workflows for [bare\_metal](../image_build/bare-metal)
 * debian - wrapper workflows for [debian](../image_build/debian)
 * enterprise\_linux - wrapper workflows for
   [enterprise\_linux](../image_build/enterprise_linux)
 * kokoro - workflow for an internal use image
-* linux\_dev - unknown/unused (?)
-* [rhui](rhui/README.md) - workflows for internal use images
 * sqlserver - wrapper workflows for [sqlserver](../image_build/sqlserver)
 * windows - wrapper workflows for [windows](../image_build/windows)
 * windows\_container - wrapper workflows for
   [windows\_container](../image_build/sqlserver)
 
+Note: At one point, [Rocky Linux](https://rockylinux.org/)
+builds were nested under `enterprise_linux` but have since been taken over by
+[CIQ](https://ciq.com/), which can be found
+[here](https://github.com/ctrliq/gcp-public-images/tree/main/daisy).
```

Comment:
I'd suggest not using [here] and instead ", which can be found at https://github.com/ctrliq/gcp-public-images/tree/main/daisy"

Reply:
Updated